src/forwarder.ts

```diff
@@ -28,12 +36,25 @@ interface DescribeSubscriptionFiltersResponse {
 function isLogGroup(value: any): value is LogGroupResource {
   return value.Type === logGroupKey;
 }
+/**
+ * Validates whether ARN exists in the account
```

Comment:
Don't forget to update the comment too 🙂 you are validating whether a Lambda function exists, not just any ARN

Reply:
And specifically a Forwarder!